fe/fe-core/src/main/java/com/starrocks/scheduler/Task.java

```diff
@@ -0,0 +1,149 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.scheduler;
+
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.common.io.Text;
+import com.starrocks.common.io.Writable;
+import com.starrocks.persist.gson.GsonUtils;
+import com.starrocks.statistic.Constants;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.util.Map;
+import java.util.concurrent.ScheduledFuture;
+
+public class Task implements Writable {
+
+    @SerializedName("id")
+    private Long id;
+
+    @SerializedName("name")
+    private String name;
+
+    @SerializedName("taskType")
+    private Constants.TaskType taskType = Constants.TaskType.MANUAL;
+
+    @SerializedName("createTime")
+    private LocalDateTime createTime;
+
+    @SerializedName("dbName")
+    private String dbName;
+
+    @SerializedName("definition")
+    private String definition;
+
+    @SerializedName("taskRunBuilder")
+    private TaskRunBuilder taskRunBuilder;
+
+    @SerializedName("properties")
+    private Map<String, String> properties;
+
+    private Schedule schedule;
+
+    private ScheduledFuture<?> future;
+
+    private TaskRunProcessor processor;
```

Comment:
I think these fileds are not stored in task.
Task is only changed when users changed, so only specified fields are stored in this class.

Reply:
I will try to move to TaskManger's field.